src/app/components/node/node.component.ts

```diff
@@ -22,102 +23,75 @@ export class NodeComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   @Input()
-  blockLabel = '';
+  displayNode: DisplayNode|null = null;
 
-  @Input()
-  blockLevel = 1;
-
-  @Input()
-  blockList: any[] = []; // TODO FIXME
-
-  @Input()
-  children: CellData[] = [];
-
-  @Input()
-  color: string = '#555';
-
-  @Input()
-  genes = 0;
-
-  @Input()
-  ignoreScaling = false;
-
-  @Input()
-  jsonData?: CellData;
-
-  @Input()
-  key = '';
-
-  @Input()
-  label = '';
-
-  @Input()
-  parent: NodeComponent|undefined;
-
-  @Input()
-  parentTotal = 0;
+  subscriptions: Subscription[] = [];
 
-  @Input()
-  sample = '';
+  consolidatedVariantInfo = getEmptyVariantInfo();
+  sortedSummaryTableRows: SummaryTableRow[] = [];
+  selectedNode: DisplayNode|null = null;
+  selectedSample: Sample|null = null;
+  showTable = false;
 
-  @Input()
-  showEndBlock = false;
+  clusterIdToCluster = new Map<number, Cluster>();
+  snvIdToDisplayVariant = new Map<number, DisplayVariant>();
 
-  @Input()
-  showStats = true;
-
-  subscriptions: Subscription[] = [];
-
-  parentsAggregate = {
-    't1_mutations': 0,
-    'mutations': 0,
-    't1_cgc_genes': 0,
-    'genes': 0,
-    'drugs': 0,
-  };
-  phylogenyProportionId: any; // TODO FIXME
-  phylogenyShowTable = false;
-  selectedBlocks: NodeComponent[] = [];
+  gradientId = 'uninitialized';
 
   summaryColumns = [
```

Comment:
chatted with Charles about what's possible with the new pipeline data and arrived at this set of columns for now